src/app/content/practiceQuestions/selectors.ts

```diff
@@ -76,3 +76,13 @@ export const isFinalQuestion = createSelector(
   currentQuestionIndex,
   (count, index) => count - 1 === index
 );
+
+export const questionsInProggress = createSelector(
+  currentQuestionIndex,
+  (index) => index !== null
+);
```

Comment:
I'm not really sure if this is needed. We can just check if `currentQuestionIndex !== null` in the component if we has to

Reply:
That's true. I've made a separate selector out of it, because it was one of the goals in the description of the [issue](https://github.com/openstax/unified/issues/1340).

Reply:
Let's leave it then